Offseason-Remade/src/main/java/frc/robot/OI.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.buttons.Button;
+import edu.wpi.first.wpilibj.buttons.JoystickButton;
+import frc.robot.Commands.ChangeLiftHeight;
+import frc.robot.Commands.ChangedDesiredHeight;
+import frc.robot.Commands.CollectCargo;
+import frc.robot.Commands.HatchLock;
+
+/**
+ * This class is the glue that binds the controls on the physical operator
+ * interface to the commands and command groups that allow control of the robot.
+ */
+public class OI {
+  Joystick driverJoystick = new Joystick(0);
+
+  Joystick operatorJoystick = new Joystick(1);
+  // operator buttons
+  Button operatorButtonA = new JoystickButton(operatorJoystick, 0);
+  Button operatorButtonB = new JoystickButton(operatorJoystick, 1);
+  Button operatorButtonX = new JoystickButton(operatorJoystick, 2);
+  Button operatorButtonY = new JoystickButton(operatorJoystick, 3);
+  Button operatorLeftBumper = new JoystickButton(operatorJoystick, 4);
+  Button operatorRightBumper = new JoystickButton(operatorJoystick, 5);
+
+  public OI() {
+
+    operatorLeftBumper.whenPressed(new ChangedDesiredHeight(-1));
+    operatorRightBumper.whenPressed(new ChangedDesiredHeight(1));
+    // TODO: find real values
```

Comment:
operatorLeftBumper.whenPressed(new ChangedDesiredHeight(-1));
operatorRightBumper.whenPressed(new ChangedDesiredHeight(1));

i have to say that this is awful way to change heights but if you do it like this at least print to the dashboard the current desiered height!!! **must**

Reply:
DID YOU FIX THAT??